src/trackers/UNIT3D/retromoviesclub.py

```
@@ -0,0 +1,203 @@
+        category_id = {"MOVIE": "1"}
+        if mapping_only:
+            return category_id
+        if reverse:
+            return {value: key for key, value in category_id.items()}
+        category_value = category or meta.category
+        return {"category_id": category_id.get(category_value, "0")}
+
+    async def get_type_id(
+        self,
+        meta: Meta,
+        type: str = "",
```

Ruff (0.16.0)

[error] Function argument `type` is shadowing a Python builtin

(A002)